parity-poa/aura-config.toml

```diff
@@ -52,7 +54,7 @@ disable = true
 
 [ipfs]
 # You won't be able to hash-query blockchain data.
-disable = true
+enable = false
```

Comment:
Same as above, comment needs to be updated.  Also, I'm curious what this actually does...

Reply:
I read a tiny bit about it, it's like since ethereum stuff is all hashes, you can export those via IPFS and serve up ethereum data via ipfs for... reasons.